.travis.yml

```diff
@@ -17,7 +17,6 @@ addons:
 
 cache:
   directories:
-  - node_modules
```

Comment:
That setting tells travis that it can cache the node_modules directories during builds. https://docs.travis-ci.com/user/caching/#Caching-directories-(Bundler%2C-dependencies). If you need to clear the cache during CI there is an option to do so on the build page under the drop-down in the upper right corner.